test/controllers/HomepageControllerSpec.scala

```diff
@@ -44,7 +44,7 @@ class HomepageControllerSpec extends FrontEndTestHelper {
       |</a>""".stripMargin
 
   val transfersForReviewButton: String =
-    """<a href="#" role="button" draggable="false" class="govuk-button" data-module="govuk-button">Transfers for Review</a>"""
+    """<a href="/admin/metadata-review" role="button" draggable="false" class="govuk-button" data-module="govuk-button">Transfers for Review</a>"""
```

Comment:
Can you add a check that the `view transfer` link should not be visible for TNA users?